### PR TITLE
unkown cmd & string fix

### PR DESCRIPTION
**Pull request by gregorybel**
- reactivate request for 2 unknown cmds
- fix initial value for limitter battery string to something general, this value is then overloaded by real device name

Successfully tested!

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java [hunk 1]
```diff
@@ -141,21 +140,9 @@ public static boolean checkBlukonPacket(byte[] buffer) {
             if (currentCommand.startsWith("810a00")) {//ACK sent
                 //ack received
 
-                if (JoH.pratelimit(BLUKON_GETSENSORAGE_TIMER, GET_SENSOR_AGE_DELAY)) {
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java]: I haven't followed the data flow in this class, are you sure about these deleted lines?

Comment (gregorybel) [reply app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java]: thanks for reviewing, yes I am quiet sure, this code portion can be Found at line 247.
no idea why this code porion was 2 times here, historical thing.

**Comment by jamorham** [comment]
With the language changes for the Libre battery, I am a little worried that with this extra text length its going to have the potential to overflow on small displays with a lot of the extra status features enabled. 

I'm thinking that the ideal change would be to make a formatted string, replacing `LimiTTer` with `%s` and then populating it from `DexCollectionType.getBestLimitterHardwareName()`

**Comment by jamorham** [comment]
Looking at the code it looks like there already is a similar workaround in place:

```
    final String limitterName = DexCollectionService.getBestLimitterHardwareName();
                    if (limitterName.equals(DexCollectionService.LIMITTER_NAME)) {
                        dexbridgeBattery.setText(getString(R.string.limitter_battery) + ": " + bridgeBattery + "%");
                    } else {
                        dexbridgeBattery.setText(limitterName + " " + getString(R.string.battery) + ": " + bridgeBattery + "%");
                    }
```

Based on that can you explain what the language changes are intended to do? As far as I can see the `limitter_battery` is only used when it is actually a LimiTTer device.

**Comment by gregorybel** [comment]
so actually maybe is my fix the wrong one.

Issue is after a fresh start, look at this picture (with the fix already), the problem is before string get updated, until a new BG value is received, xdrip shows the default string. Thats why I tought we need to change default string.

**Comment by gregorybel** [comment]
I can also reduce string to "Libre battery" and like this we need less characters than before.

**Comment by jamorham** [comment]
I reckon we just change it to Libre Battery and after the first connection to the device it will show the correct name.

**Comment by gregorybel** [comment]
I just changed to "Libre", fine for you?

**Comment by jamorham** [comment]
Looks good to me, I will merge

**Comment by gregorybel** [comment]
thank you

**Pull request merged**